puppet/modules/chassis/manifests/wp.pp

```diff
@@ -11,6 +11,8 @@
 	$admin_email    = 'admin@example.com',
 	$admin_password = 'password',
 	$network = false,
+	$plugins = [],
+	$theme = '',
```

Comment:
I'm thinking it might be good to do an array of themes as well? The rationale behind that is that sometimes with multisite you'll have a few different themes and not just one. We could perhaps make it so the first of the themes listed in the `.yaml` file is `enabled`. What are your thoughts Sir?